circuit-codegen/src/main/kotlin/com/slack/circuit/codegen/CircuitSymbolProcessorProvider.kt

```diff
@@ -409,10 +416,16 @@ private class CircuitSymbolProcessor(
             }
             .firstOrNull()
             ?: run {
+              val annotationsString =
+                declaration.annotations.toList().joinToString {
+                  it.annotationType.resolve().toTypeName().toString()
+                }
               logger.error(
                 "Factory must be for a UI or Presenter class, but was " +
-                  "${targetClass.qualifiedName?.asString()}. " +
-                  "Supertypes: ${targetClass.getAllSuperTypes().toList()}",
+                  "${targetClass.qualifiedName?.asString()}.\n" +
+                  "Supertypes: ${targetClass.getAllSuperTypes().toList()}.\n" +
+                  "isAssisted? ${isAssisted}\n" +
+                  "Annotations: $annotationsString}",
```

Comment:
This extra logging was helpful to figure out that KSP couldn't see through the actualized typealias so opted to leave it in